logic/src/commonMain/kotlin/com/wire/kalium/logic/KaliumScope.kt

```diff
@@ -0,0 +1,63 @@
+package com.wire.kalium.logic
+
+import com.wire.kalium.logic.configuration.UserConfigDataSource
+import com.wire.kalium.logic.configuration.UserConfigRepository
+import com.wire.kalium.logic.configuration.notification.NotificationTokenDataSource
+import com.wire.kalium.logic.configuration.notification.NotificationTokenRepository
+import com.wire.kalium.logic.configuration.server.ServerConfigDataSource
+import com.wire.kalium.logic.configuration.server.ServerConfigRepository
+import com.wire.kalium.logic.data.session.SessionRepository
+import com.wire.kalium.logic.feature.auth.AddAuthenticatedUserUseCase
+import com.wire.kalium.logic.feature.notificationToken.SaveNotificationTokenUseCase
+import com.wire.kalium.logic.feature.server.GetServerConfigUseCase
+import com.wire.kalium.logic.feature.server.ObserveServerConfigUseCase
+import com.wire.kalium.logic.feature.server.UpdateApiVersionsUseCase
+import com.wire.kalium.logic.feature.server.UpdateApiVersionsUseCaseImpl
+import com.wire.kalium.logic.feature.session.GetSessionsUseCase
+import com.wire.kalium.logic.feature.session.SessionScope
+import com.wire.kalium.logic.feature.user.EnableLoggingUseCase
+import com.wire.kalium.logic.feature.user.EnableLoggingUseCaseImpl
+import com.wire.kalium.logic.feature.user.IsLoggingEnabledUseCase
+import com.wire.kalium.logic.feature.user.IsLoggingEnabledUseCaseImpl
+import com.wire.kalium.network.UnboundNetworkContainer
+import com.wire.kalium.persistence.client.TokenStorage
+import com.wire.kalium.persistence.client.TokenStorageImpl
+import com.wire.kalium.persistence.client.UserConfigStorage
+import com.wire.kalium.persistence.client.UserConfigStorageImpl
+import com.wire.kalium.persistence.db.GlobalDatabaseProvider
+import com.wire.kalium.persistence.kmm_settings.KaliumPreferences
+
+class KaliumScope(
```

Comment:
`getGlobalScope` returns a `kaliumScope`.

Not really terrible, considering Coroutine's `GlobalScope`.

What if we just make it `GlobalKaliumScope` so there are NO questions about what it is?


Also, in CASE there are questions about it for people getting into the project. We do have Dokka :)

Could you add some docs about the finality of this Scope, and mention how it differs from others?

Like this (requires import of `AuthenticationScope` and `UserSessionScope`:

```kotlin
/**
  * Scope that exposes all operations that are user and backend agnostic, like
  * - Storing and retrieving sessions after authenticating
  * - Updating client or device metadata (like push notification token)
  * - Getting back-end information from a deeplink
  *
  * @see AuthenticationScope
  * @see UserSessionScope
  */
class KaliumScope(
```